roles/installer/tasks/main.yml

```diff
@@ -36,3 +36,6 @@
   register: foreman_installer_run
   changed_when: foreman_installer_run.rc == 2
   failed_when: foreman_installer_run.rc not in [0, 2]
+  environment:
+    LC_ALL: "{{ foreman_installer_locale | default('en_US.UTF-8') }}"
```

Comment:
I think the default should be defined in `roles/installer/defaults/main.yml` and documented in `README.md`

Reply:
Done!

Reply:
Done!